easybuild/easyconfigs/s/squashfs/squashfs-4.3-GCC-5.4.0-2.26.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = 'ConfigureMake'
+
+name = 'squashfs'
+version = '4.3'
+
+homepage = 'https://sourceforge.net/projects/squashfs'
+description = """Squashfs is a highly compressed read-only filesystem for Linux."""
+toolchain = {'name': 'GCC', 'version': '5.4.0-2.26'}
+
+source_urls = ['http://sourceforge.net/projects/squashfs/files/squashfs/squashfs%(version)s/']
+sources = ['%(name)s%(version)s.tar.gz']
+checksums = ['0d605512437b1eb800b4736791559295ee5f60177e102e4d4ccd0ee241a5f3f6']
+
+start_dir = 'squashfs-tools'
+skipsteps = ['configure']
+installopts = 'INSTALL_DIR=%(installdir)s'
```

Comment:
@bilke This should probably be tweaked to `%(installdir)s/bin`, to ensure that `mksquashfs` and `unsquashfs` are in a standard location that gets included into `$PATH`?

Cfr expectations for the `--with-mksquashfs` configure option that's being added to Singularity: https://github.com/singularityware/singularity/pull/1530